<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COMP_hack Logger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="logEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>368</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuStart_Game">
     <property name="title">
      <string>Start Game</string>
     </property>
     <addaction name="actionAdd_client"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="action_Settings"/>
    <addaction name="separator"/>
    <addaction name="menuStart_Game"/>
    <addaction name="action_Captures"/>
    <addaction name="separator"/>
    <addaction name="actionShutdown"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_Settings">
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Captures">
   <property name="text">
    <string>&amp;Captures...</string>
   </property>
  </action>
  <action name="actionShutdown">
   <property name="text">
    <string>Shutdown</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAdd_client">
   <property name="text">
    <string>Add client...</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
